portfolio/src/main/webapp/scripts/github/index.js

```diff
@@ -0,0 +1,48 @@
+import { github } from "./client.js";
```

Comment:
nit: I'd probably also use single quotes here to match the examples in the style guide (https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes)

Same in other files